src/StdUtils.sol

```diff
@@ -4,12 +4,12 @@ pragma solidity >=0.6.2 <0.9.0;
 import "./console2.sol";
 
 abstract contract StdUtils {
+    uint256 private constant INT256_MIN_ABS =
```

Comment:
_(arbitrary comment location to start a thread)_ 

> One problem is that `log(string,int256)` does not exist so I used `log(string,uint256)` instead of it.

If you want to add support for this:
1. Add the required method to `console2.sol` in forge-std
2. Update the [ABI file](https://github.com/foundry-rs/foundry/blob/master/evm/abi/console.json) in foundry

I think that should be all that's needed, but I may be missing something. Also related is https://github.com/foundry-rs/forge-std/issues/129 if you wanted to add that overload at the same time

Reply:
Okay, I'll check this out.

Reply:
What will be the better one, `log(string,int256)` or `logInt(string,int256)`?
At the [related thread](https://github.com/NomicFoundation/hardhat/issues/1440) of #129, they decided to name `logInt(int256...)` rather than `log(int256...)` because of the ambiguity.

Reply:
I'm ok with `log(string,int256)` because in practice you'll be doing `console.log(myVar)` (unambiguous since `myVar` has a type) instead of `console.log(3)` (ambiguous because we don't know 3's type)

Reply:
Applied at [here](https://github.com/foundry-rs/forge-std/pull/253/commits/3e50c5d244cdfa055aa6468334ad6193e746d7fe)!

Reply:
Thanks! Note that the logs won't actually be shown until we upstream the corresponding change to forge. I can do that soon though